simulation.py

```diff
@@ -143,12 +143,17 @@ def simulate(results_queue, data_all, alpha_settings_path, skip_evaluation):
     pnl_function = TRADING_CONFIGS[settings.target][PNL_FUNCTION](
         OFFICIAL_CURRENCY[region], TRADING_INSTRUMENTS[settings.target])
     results = Simulator(settings, data_required, pnl_function).run()
+    eval_url = 'http://127.0.0.1:5000/'
+    eval_data = {
+        'indicator_names': None,
+        'instruments': TRADING_INSTRUMENTS[settings.target],
+        'data': results}
     eval_results = (
-        Evaluator().run(TRADING_INSTRUMENTS[settings.target], results)
+        requests.post(eval_url, data=pickle.dumps(eval_data)).content
         if not skip_evaluation else None)
     results_queue.put(
-        (alpha_settings_path, TRADING_INSTRUMENTS[settings.target],
-         results, eval_results))
+        (alpha_settings_path, TRADING_INSTRUMENTS[settings.target], results,
+         pickle.loads(eval_results) if eval_results else None))
```

Comment:
It is possible that evaluation server is not available or down when we send our requests.
Can you handle such exceptions here?